content/en/catalyst/fund9/challenges/the-great-migration-eth/campagin-brief.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "The Great Migration (from Ethereum)"
+linkTitle: "The Great Migration (from Ethereum)"
+type: docs
+hide_feedback: true
+description: "How might we make it easy and fast for dApps and other projects to expand or completely move from Ethereum to Cardano?"
+---
+<!-- <img src="https://cardano.ideascale.com/community-library/accounts/93/936143/Public/21-Community-Advisor-Improvements-dbdbe2.png" style="width:500px;height500px"> -->
+
+Fund Size: **500,000 USD paid in Ada**
+
+[Campaign page](https://cardano.ideascale.com/c/idea/405419) and [browse ideas]().
+
+## Challenge Brief
+The Cardano ecosystem, despite being written in Haskell, has always had plans to provide a path for projects that either deployed on an EVM-based blockchain (like Ethereum) to migrate to the Cardano ecosystem. Additionally, projects that are better suited for the account-style method (instead of UTXO) also exist, and we want these projects to be usable from the Cardano ecosystem as well.
+
+To accomplish this part of the roadmap, many efforts are currently underway such as KEVM, IELE, ERC20 Converter and Milkomeda to name a few. These projects are reaching the point where they are mature enough for people to deploy using them or build tooling for them and so it's time to start welcoming projects to Cardano.
+
+You can hear Charles talk about the importance of this in his video [The Island, The Ocean and the Pond](https://www.youtube.com/watch?v=k8a6tX53YPs)
+
+### Why is it important?
+Many projects are stuck on EVM-based blockchains like Ethereum and are looking for an escape. Cardano can be faster and cheaper to use.
+
+### What does success look like?
+Projects who were either planning to or did launch on an EVM-based blockchain deploy in the Cardano ecosystem and can do so easily.
+
+### Key Metrics to measure
```

Comment:
'Key Metrics to measure' contents should be:
   - Number of projects that migrate
   - Number of users of these smart contracts once migrated to the Cardano ecosystem
(Metrics can either be improved by funding projects that directly deploy something, or projects that help build tooling to make it easier to deploy)